pkg/operator/k8sutil/serviceaccount.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2024 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package k8sutil
+
+// DefaultServiceAccount is a  service-account used for components that do not specify a dedicated service-account.
+const DefaultServiceAccount = "rook-ceph-default"
```

Comment:
nit: How about defining this const in k8sutil.go? IMO it's not usually necessary to create a new go file if there is somewhere else to include something small like this.

Reply:
Updated